services.qmd

```diff
@@ -3,57 +3,61 @@ title: "Services"
 
-This draws on experience refactoring messy internals, simplifying logic, adding CI, and reviewing open source contributions for Apache Arrow. The goal is always to improve robustness and maintainability without overengineering.
 ---
 
-### **3. Migrating from Legacy Tools to Open Source R Workflows**
+### **3. Planning and Delivering Migrations to R from SAS, Stata, or Excel**
+
+Tools like SAS, Stata, or Excel still underpin a lot of internal workflows — but they’re often hard to scale, audit, or maintain. I help teams plan and deliver migrations to R that are practical, phased, and grounded in how the tools are actually used.
 
-Tools like Excel, SAS, or Stata are still widely used, but can be limiting - expensive, inflexible, and hard to integrate. I help teams explore and plan a move to R: starting with an audit of existing code and workflows, then identifying what can be translated directly, what needs rethinking, and what risks to watch for.
+That starts with scoping: reviewing existing scripts, mapping dependencies, and figuring out what can be translated directly versus what needs redesign. From there, I can help implement the migration — reverse-engineering legacy logic, rebuilding it in R, and setting things up for long-term use.
 
-I start with a review of your existing code and workflows to map what can be converted easily, what needs redesign, and where risks sit. One past project involved analysing a large SAS codebase to scope out a realistic migration path — surfacing complexity early to avoid costly surprises later.
+This kind of work is a good fit when the scope is clear and the migration needs to be done carefully, not just quickly. I’ve done this for research and reporting pipelines, including analysis of large SAS-based systems, and know how to handle mess and ambiguity without getting stuck in it.
 
 ---
 
 ### **4. General R Consulting and Team Support**
 
-Not everything fits into a box. Sometimes you need an outside perspective on a flaky pipeline, help shaping a new workflow, or someone to scope a project before committing dev time.
+Not everything fits into a neat category. Sometimes you need a second pair of eyes on a fragile pipeline, help shaping a new workflow, or someone to make sense of a half-built tool before you commit more dev time.
 
-I also offer short advisory engagements, mentoring for internal developers, and lightweight architecture or code reviews. These can be standalone or follow on from an audit or build.
+I offer flexible support for teams working in R — from lightweight code or architecture reviews to mentoring and advisory sessions. This can be a one-off chat, a review of a specific part of a codebase, or an ongoing check-in as things evolve.
```

Comment:
```suggestion
I offer flexible support for teams working in R; from lightweight code or architecture reviews to mentoring and advisory sessions. This can be a one-off chat, a review of a specific part of a codebase, or an ongoing check-in as things evolve.
```